use/using-the-app.md

```diff
@@ -14,83 +14,86 @@ The examples here have a blank background for simplicity, but you will see all o
 
 ## Navigating the Menus
 
-![intro to the menus](./images/ui-tutorial-gifs/01-intro-to-the-menus.gif)
-
+<img width="60%" src="./images/ui-tutorial-gifs/01-intro-to-the-menus.gif" style="float: left; margin-right: 20px; margin-bottom: 20px;" />
 There are five menu buttons always present on the right side of the screen. Learning what each does will guide you through the interface.
+<p style = "clear: left;">
 
 - The **Interfaces** button displays any AR *tools* placed into the space.
 - The **Programming** button displays the programming *nodes* and *links* for any of the tools you are currently looking at.
 - The **Pocket** button opens a scrolling menu of tools that you can add to the space.
 - The **Settings** button opens the settings menu, where you toggle between different options and modes.
 - The **Freeze** button is a useful feature to pause the video background into a still image, so the AR content doesn't move when you move your device.
 
+</p>
+
 ## The Basics
 
 ### Adding Tools
 
-![add slider and graph from pocket](./images/ui-tutorial-gifs/16-adding-tools-from-pocket.gif)
+<img width="60%" src="./images/ui-tutorial-gifs/16-adding-tools-from-pocket.gif" align="left" style="margin-right: 20px; margin-bottom: 30px;" /> 
 
 Tapping on an icon in the pocket will add that *tool* to the space.
 
 You can then tap and hold on the tool for a moment (without moving your finger) to start moving it. Drag to move it horizontally and vertically. Pinch with two fingers to scale it up or down.
 
 In this example, we add a slider tool and a graph tool, and organize them in our space.
+<p style = "clear: left;"></p>
 
 ### Drawing Links
 
-![draw link from left node to right node](./images/ui-tutorial-gifs/02-create-link.gif)
+<img width="60%" src="./images/ui-tutorial-gifs/02-create-link.gif" align="left" style="margin-right: 20px; margin-bottom: 30px;" /> 
 
 When you switch to programming mode, you will see the *nodes* for any visible tools.
 
 You can connect any two nodes by dragging a line between them.
 
-This creates a *link* that will send data from the first node (in this example, on the left) to the second (on the right). The direction matters, so the animated dots move in the direction that data will flow.
+This creates a *link* that will send data from the first node (in this example, on the left) to the second (on the right). The direction matters, so the animated dots move in the direction that data will flow.<p style = "clear: left;"></p>
```

Comment:
What's the purpose of all the clear: lefts? It looks like these should all already be line breaks